tutorials/source-modeling/plot_mne_solutions.py

```diff
@@ -30,6 +30,7 @@
 fname_cov = data_path + '/MEG/sample/sample_audvis-cov.fif'
 fwd = mne.read_forward_solution(fname_fwd)
 cov = mne.read_cov(fname_cov)
+evoked.crop(0.05, 0.15, verbose='error')  # ignore baseline
```

Comment:
General question: Is it actually a good idea to simply ignore the baseline when examining the inverse solution? Is it not useful to know what the baseline looks like? At the sensor level, it's extremely valuable information… and I always thought it helps me decide whether things are going well on the source level as well. So, if normally one should **also** look at the inverse baseline, we should explicitly state that we're merely cropping here to save some memory and suggest users "do not try this at home".

Reply:
There are many things it's useful to investigate at many stages, I don't think we need to show / include them in all examples